easybuild/easyconfigs/x/xcb-proto/xcb-proto-1.11.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'xcb-proto'
+version = '1.11'
+
+homepage = 'http://xcb.freedesktop.org/'
+description = """The X protocol C-language Binding (XCB) is a replacement for Xlib featuring a small footprint,
+latency hiding, direct access to the protocol, improved threading support, and extensibility."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://xcb.freedesktop.org/dist/']
+sources = [SOURCELOWER_TAR_GZ]
+
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+pyshortver = '.'.join(SYS_PYTHON_VERSION.split('.')[0:2])
+
+sanity_check_paths = {
+    'files': ['lib/pkgconfig/xcb-proto.pc'],
+    'dirs': ['lib/python%s/site-packages/xcbgen' % pyshortver]
```

Comment:
this is an issue...

if the system Python is updated from 2.6 to 2.7, this will not be found anymore?

besides, this is not in `$PYTHONPATH` anyway, as it is now (same issue when we include our own Python version)... 

does that mean the Python bits are basically irrelevant?